libevm/VM.cpp

```diff
@@ -228,8 +263,8 @@ void VM::interpretCases()
         CASE(CREATE2)
         {
             ON_OP();
-            if (!m_schedule->haveCreate2)
-                throwBadInstruction();
+            // TODO: Bring back support for CREATE2.
```

Comment:
How about `m_rev < EVM_CONSTANTINOPLE` (if it is still planned for that)?

Reply:
I'm not sure it's planned. We will enable this when the decision is made.